solr/solr-ref-guide/build.gradle

```diff
@@ -90,14 +90,14 @@ dependencies {
     depVer('org.apache.zookeeper:zookeeper')
 
     // jekyll dependencies
-    gems 'rubygems:jekyll:3.5.2'
+    gems 'rubygems:jekyll:4.1.1'
```

Comment:
There are significant changes between Jekyll 3 and 4 and we haven't upgraded to it yet because it needs some careful analysis and fixing. In the past attempts to build with Jekyll 4 have failed. I don't see evidence in this PR that you have done sufficient testing for me to feel confident that this is OK.

Reply:
Do you remember any specific examples I could double-check on? I saw some past discussions, but it was about ant and PDF issues. I can double check. However, from the analysis I did do, only source blocks were actually affected.